spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-lattice-core</artifactId>
 			<version>${spring-cloud-lattice.version}</version>
+			<scope>runtime</scope>
```

Comment:
Why is 'runtime' scope needed? I've been running the admin on Lattice and the "lattice" and "cloud" profiles have activated, before this PR.